### DISCUSSION TITLE
Japan unusual terrain slope

### DISCUSSION THREAD

**Discussion by Jordan886** [kubilus1/autoortho#214] [General]
Hi,
first of all, this is so far one of the best thing ever created for XP.

I'm using it all around Europe and so far i had no problems.
I flew in japan and I noticed that Haned (RJTT) and Fukuoka (RJFF) have strange hills on the runway that are unrealistic and problematic for takeoff and landing.

Is it a bug or I'm doing something wrong?
Also, sometimes i find some minor bug in Europe (like a huge strange terrain "depression" on the border between austria and poland), is this the right place to report them?

Thanks!

**Comment by kubilus1** [comment]
Thanks, yeah there is probably something with how the terrain is generated.  I'll take a look when I can.
